dev/README.md

```diff
@@ -62,14 +63,42 @@ This will spin up the following services:
 - `sdp-api`: SDP service running on port `8000`.
 - `sdp-tss`: Transaction Submission service.
 - `sdp-frontend`: SDP frontend service running on port `3000`.
+- `kafka`: Kafka service running on port `9092`.
+- `kafka-init`:  Initial workflow to exec into the Kafka container and create topics.
+- `db-conduktor`: Conduktor service.
+- `conduktor-monitoring`: Conduktor Monitoring service integrated into the Conduktor Platform.
+- `conduktor-platform`: Provides solutions for Kafka management, testing, monitoring, data quality, security, and data governance.
+
+
+### New Tenant Provisioning Process
+
+To provision new tenants, you must run `main.sh`, which will create `tenants=("redcorp" "bluecorp")`. 
+
+```
+./main.sh
+```
+
+Be sure that both hosts are included in the host configuration file.
+To check it, you can run the command `cat /etc/hosts`.
+To include them, you can run command `sudo nano /etc/hosts` and insert the lines below:
+
+```
+127.0.0.1       bluecorp.sdp.local
+127.0.0.1       redcorp.sdp.local
+```
 
 ### Create an Owner SDP User
 
 Open a terminal for the `sdp-api` container and run the following command to create an owner user:
 
 ```sh
 docker exec -it sdp-api bash # Or use Docker Desktop to open terminal
-./stellar-disbursement-platform auth add-user owner@stellar.org joe yabuki --password --owner --roles owner
+./stellar-disbursement-platform auth add-user owner@stellar.org joe yabuki --password --owner --roles owner --tenant-id TENANT_ID
```

Comment:
Can we add a note that is running `./main.sh` already adds a user to the database? This is in case the user wants another user.

Reply:
Thanks!

Reply:
I think we should replace this step all together `### Create an Owner SDP User`

Instead we should have a step for `### Setup Owner User Password for each tenant`. 

This step would guide users to go through the forgot password flow for the owner `john.doe@bluecorp.org` or `john.doe@redcorp.org`. 

Forgot Password: `http://bluecorp.stellar.local:3000/forgot-password`
Reset Password: `http://bluecorp.stellar.local:3000/reset-password`

Once they do that, they should be able to login.